src/controls/ProgressStepsIndicator/IProgressStepsIndicatorProps.ts

```diff
@@ -0,0 +1,9 @@
+import { IReadonlyTheme } from '@microsoft/sp-component-base';
+
+import { IStep } from './IStep';
+
+export interface IProgressStepsIndicatorProps {
+  steps?:IStep[];
```

Comment:
Can this property be optional? Can we have the control without steps?

Reply:
fixed